openmetadata-ui/src/main/resources/ui/src/components/common/DisplayName/DisplayName.tsx

```diff
@@ -59,44 +56,40 @@ const DisplayName: React.FC<DisplayNameProps> = ({
 
   return (
     <div className="d-inline-flex flex-column hover-icon-group w-max-90">
-      <div className="d-inline-flex items-baseline">
-        <Typography.Text
-          className="m-b-0 d-block text-grey-muted break-word"
-          data-testid="column-name">
-          {isEmpty(displayName) ? (
-            <Link className="break-word" data-testid={name} to={link}>
-              {name}
-            </Link>
-          ) : (
-            name
-          )}
-        </Typography.Text>
-      </div>
-      {!isEmpty(displayName) ? (
-        // It will render displayName fallback to name
-        <Typography.Text
-          className="m-b-0 d-block break-word"
-          data-testid="column-display-name">
+      <Typography.Text
+        className="m-b-0 d-block text-grey-muted break-word"
+        data-testid="column-name">
+        {isEmpty(displayName) ? (
           <Link className="break-word" data-testid={name} to={link}>
-            {displayName}
+            {name}
           </Link>
-        </Typography.Text>
-      ) : null}
+        ) : (
+          <>
+            {name}
+            <Typography.Text
+              className="m-b-0 d-block break-word"
+              data-testid="column-display-name">
+              <Link className="break-word" data-testid={displayName} to={link}>
+                {displayName}
+              </Link>
+            </Typography.Text>
+          </>
+        )}
+      </Typography.Text>
 
       {allowRename ? (
         <Tooltip placement="right" title={t('label.edit')}>
           <Button
-            className="cursor-pointer hover-cell-icon w-fit-content"
+            className="cursor-pointer hover-cell-icon w-fit-content display-name-edit-button"
```

Comment:
We can eliminate these styling classes and unify all stylings in the `display-name-edit-button` class